Containers/Login/Login.js

```diff
@@ -1,39 +1,92 @@
 import React, {Component} from 'react';
-import { StyleSheet, Text, View, TouchableOpacity, ImageBackground } from 'react-native';
+import { StyleSheet, Text, View, TouchableOpacity, ImageBackground, TextInput } from 'react-native';
 import { Icon } from 'react-native-elements';
 
 export class Login extends Component {
+    constructor() {
+        super()
+        this.state = {
+            username: '',
+            password: '',
+            confirmPassword: ''
+        }
+    }
     render() {
-        return (
-            <View style={styles.container}>
+        if (this.props.isLoggedIn) {
+            return ( 
+                <View style={styles.container}>
                 <ImageBackground source={require('../../assets/goldengate.jpg')} style={{width: '100%', height: '100%', display: 'flex', flex: 1, flexDirection: 'column', justifyContent: 'space-evenly', alignItems: 'center'}}>
                 <View style={styles.overlay} />
-                <View>
-                    <Icon color="white" name="user-circle" type="font-awesome" size={90} padding={15} top={30}/>
-                </View>
-                <View>
-                    <TouchableOpacity style={{ backgroundColor: '#e9e9e9', width: 200, padding: 10, borderRadius: 15, bottom: 10 }}>
-                        <Text>Create account</Text>
```

Comment:
For some reason everything is centered except the "Create account".  I can't see where the difference is, going to approve PR and put this as an issue on our waffle board.